pkg/ccl/streamingccl/streamingest/stream_ingestion_job.go

```diff
@@ -182,19 +197,20 @@ func ingest(ctx context.Context, execCtx sql.JobExecContext, ingestionJob *jobs.
 		startTime = *h
 	}
 
-	// If there is an existing stream ID, reconnect to it.
-	streamID := streaming.StreamID(details.StreamID)
 	// Initialize a stream client and resolve topology.
 	client, err := connectToActiveClient(ctx, ingestionJob)
 	if err != nil {
 		return err
 	}
 	ingestWithClient := func() error {
+		streamID := streaming.StreamID(details.StreamID)
+		updateRunningStatus(ctx, ingestionJob, fmt.Sprintf("connecting to the producer job %d", streamID))
```

Comment:
We might want to have just one running status that covers both the connecting and planning.  Updating the job status isn't free, so let's keep it to the highest value ones.